src/types/inline-configs.ts

```diff
@@ -22,8 +22,13 @@ export interface InlineNumberConfig extends InlineBaseConfig, InputLengthTestabl
 
 export interface InlineTextareaConfig extends InlineBaseConfig, TextareaConfig, InputRegexTestable { }
 
+export interface InlineCheckboxConfig {
```

Comment:
`InlineCheckboxConfig` should be like `InlineTextareaConfig`. It should extends from `InlineBaseConfig` and `CheckboxConfig` (importing from `src/types/input-configs.ts`. And I think it is the last request to merge it! 👍